rocAL/rocAL/source/master_graph.cpp

```diff
@@ -939,7 +939,10 @@ void MasterGraph::output_routine()
                         }
                         else
                         {
+                            if(!_is_keypoint)
+                            {
                             _meta_data_graph->update_meta_data(_augmented_meta_data, decode_image_info);
+                            }
```

Comment:
why are we not updating the meta data if its key point? 
Is it just for the PR ?

Reply:
Removed the api call since not needed